src/app/tabs.component.test.js

```diff
@@ -0,0 +1,22 @@
+describe('TabsCtrl', () => {
+  let $componentController;
+  let $rootScope;
+
+  beforeEach(angular.mock.module('AngularJSApp'));
+  beforeEach(
+    angular.mock.inject((_$componentController_, _$rootScope_) => {
+      $componentController = _$componentController_;
+      $rootScope = _$rootScope_;
+    })
+  );
+
+  it('should be defined', () => {
+    const ctrl = $componentController('appTabs');
+    expect(ctrl.$digestCount).toBe(0);
+
+    $rootScope.$digest();
+    // Expect `2`, because AngularJS always runs on more digest at the end to
```

Comment:
```suggestion
    // Expect `2`, because AngularJS always runs one more digest at the end to
```

My bad. I had the same typo in [my comment][1] 😇

[1]: https://github.com/xlts-dev/angularjs-webpack-babel/pull/18#discussion_r1138760713